flight-reservation/screens/SignUpScreen.js

```diff
@@ -0,0 +1,59 @@
+import { StyleSheet, Text, TextInput, View, Platform, StatusBar } from "react-native";
+import React from "react";
+import { TextInputComponent, ButtonComponent, CheckBoxComponent } from "../components";
+
+const SignUpScreen = () => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>Sign Up</Text>
+      <TextInputComponent title="First name"/>
+      <TextInputComponent title="Email *"/>
+      <TextInputComponent title="Password *" hint={'Use 8 or more characters with a mix of letters, numbers, and symbols.'}/>
+
+      <CheckBoxComponent text='I agree to the Terms and Privacy Policy'/>
+      <CheckBoxComponent text='Subscribe for select product updates.'/>
+
+      <ButtonComponent title='Sign Up' textColor='#fff' bgColor='gray'/>
+      <View style={styles.orSection}>
+        <Text>or</Text>
+      </View>
+      <ButtonComponent title='Sign Up with Google' textColor='#fff' bgColor='gray' icon="google"/>
+      <View style={styles.loginSection}>
+        <Text style={{color:'grey'}}>Already have an account?</Text>
+        <Text style={{color:'blue',textDecorationLine:'underline'}}>Log In</Text>
```

Comment:
Don't leave styles inside the component:

```suggestion
        <Text style={styles.ColorStyles}>Already have an account?</Text>
        <Text style={styles.LinkStyles}>Log In</Text>
```

Reply:
Addressed (: